### DISCUSSION TITLE
v2.9.8 quick fix

### DISCUSSION THREAD

**Discussion by davidm40** [LoicVeirman/HardenAD#106] [Tips]
Bonjour,
Apparemment la 2.9.8 QuickFix vient remplacer la 2.9.8, cependant je ne comprends pas pourquoi la configuration via le GUI ne fait pas partie de la version QuickFix. Du coup, faut-il télécharger la 2.9.8 et écraser le contenu avec la QuickFix ?

**Comment by LoicVeirman** [comment]
Bonjour,

Non - la quickfix EST la dernière version, comme cela est indiquée dans la note de présentation.
